docs/javascript-typescript-interop.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Blazor JavaScript and TypeScript interop
+author: danroth27
+description: Learn how to invoke JavaScript functions from .NET and .NET methods from JavaScript.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 06/10/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/javascript-typescript-interop
+---
+# Blazor JavaScript and TypeScript interop
+
+By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+A Blazor app can invoke JavaScript functions from .NET and .NET methods from JavaScript code.
+
+## Invoke JavaScript functions from .NET methods
+
+There are times when Blazor .NET code is required to call a JavaScript function. For example, a JavaScript call can expose a platform-specific capability to the Blazor app.
```

Comment:
> For example, a JavaScript call can expose a platform-specific capability to the Blazor app.

For example, a JavaScript call can expose browser capabilities or functionality from a JavaScript library to the Blazor app.